project.yaml

```diff
@@ -557,18 +557,42 @@ network:
 dataSources:
   - name: main
     kind: substrate/Runtime
-    startBlock: 1000000
+    startBlock: 	32490
     mapping:
       handlers:
-        - handler: handleXYKPools
-          kind: substrate/BlockHandler
+        
+        # - handler: handleXYKPools
+        #   kind: substrate/BlockHandler
+
+        - handler: handlerTransfers
+          kind: substrate/EventHandler
+          filter:
+            module: assets
+            method: Transfer
+        
         - handler: handleSwaps
           kind: substrate/CallHandler
           filter:
             module: liquidityProxy
             method: swap
-        - handler: handlerTransfers
-          kind: substrate/EventHandler
+        
+        - handler: handleLiquidityDeposit
+          kind: substrate/CallHandler
           filter:
-            module: assets
-            method: Transfer
+            module: poolXyk
+            method: depositLiquidity
+
+        - handler: handleLiquidityRemoval
+          kind: substrate/CallHandler
+          filter:
+            module: poolXyk
+            method: withdrawLiquidity
+        
+        - handler: handlerIrohaMigration
+          kind: substrate/CallHandler
+          filter:
+            module: irohaMigration
+            method: migrate
+        
+        # - handler: handleHistoryElement
```

Comment:
How are you going to handler other types of extrinsics?

Reply:
Other extrinsic types are not handled. Once new extrinsics start being used and the need for a new history element type appears, the new handler for a new extrinsic will be added.